input/2D/relaxation/distributed/poly_mps_ic_onestep_parallel.i

```diff
@@ -0,0 +1,336 @@
+
+[AuxKernels]
+  [./bnds_aux]
+    type = BndsCalcAux
+    variable = bnds
+    execute_on = 'initial timestep_end'
+  [../]
+  [./time]
+    type = FunctionAux
+    variable = time
+    function = 't'
+  [../]
+
+  #additional_for_grain_tracker
+  [./unique_grains]
+    type = FeatureFloodCountAux
+    variable = unique_grains
+    flood_counter = grain_tracker
+    field_display = UNIQUE_REGION
+  [../]
+  [./var_indices]
+    type = FeatureFloodCountAux
+    variable = var_indices
+    flood_counter = grain_tracker
+    field_display = VARIABLE_COLORING
+  [../]
+  [./halos]
+    type = FeatureFloodCountAux
+    variable = halos
+    flood_counter = grain_tracker
+    field_display = HALOS
+    execute_on = 'initial timestep_end'
+  [../]
+  [./halo0]
+    type = FeatureFloodCountAux
+    variable = halo0
+    map_index = 0
+    flood_counter = grain_tracker
+    field_display = HALOS
+  [../]
+  [./halo1]
+    type = FeatureFloodCountAux
+    variable = halo1
+    map_index = 1
+    flood_counter = grain_tracker
+    field_display = HALOS
+  [../]
+  [./halo2]
+    type = FeatureFloodCountAux
+    variable = halo2
+    map_index = 2
+    flood_counter = grain_tracker
+    field_display = HALOS
+  [../]
+  [./halo3]
+    type = FeatureFloodCountAux
+    variable = halo3
+    map_index = 3
+    flood_counter = grain_tracker
+    field_display = HALOS
+  [../]
+  [./halo4]
+    type = FeatureFloodCountAux
+    variable = halo4
+    map_index = 4
+    flood_counter = grain_tracker
+    field_display = HALOS
+  [../]
+  [./halo5]
+    type = FeatureFloodCountAux
+    variable = halo5
+    map_index = 5
+    flood_counter = grain_tracker
+    field_display = HALOS
+  [../]
+  [./halo6]
+    type = FeatureFloodCountAux
+    variable = halo6
+    map_index = 6
+    flood_counter = grain_tracker
+    field_display = HALOS
+  [../]
+  [./halo7]
+    type = FeatureFloodCountAux
+    variable = halo7
+    map_index = 7
+    flood_counter = grain_tracker
+    field_display = HALOS
+  [../]
+  [./halo8]
+    type = FeatureFloodCountAux
+    variable = halo8
+    map_index = 8
+    flood_counter = grain_tracker
+    field_display = HALOS
+  [../]
+  [./halo9]
+    type =  FeatureFloodCountAux
+    variable = halo9
+    map_index = 9
+    flood_counter = grain_tracker
+    field_display = HALOS
+  [../]
+  [./halo10]
+    type =  FeatureFloodCountAux
+    variable = halo10
+    map_index = 10
+    flood_counter = grain_tracker
+    field_display = HALOS
+  [../]
+[]
+
+[ICs]
+  [./PolycrystalICs]
+    [./PolycrystalVoronoiVoidIC]
+      polycrystal_ic_uo = voronoi
+      invalue = 1.0
+      outvalue = 0.0
+      numbub = 8
+      bubspac = 4000
+      radius = 400
+      profile = TANH
+    [../]
+  [../]
+  [./c_IC]
+    type = PolycrystalVoronoiVoidIC
+    variable = bubble
+    structure_type = voids
+    polycrystal_ic_uo = voronoi
+    invalue = 1.0
+    outvalue = 0.0
+    numbub = 8
+    bubspac = 4000
+    radius = 400
+    profile = TANH
+  [../]
```

Comment:
I am curious about that can we reduce the input requirement here? We have typed in the same information twice

Reply:
These can be moved to `GlobalParams`.